user/protocols-tractor.md

```diff
@@ -24,12 +27,33 @@
 
 ## Methods
+The phenotractor was equipped with three types of sensors for measuring plant height, temperature and canopy spectral reflectance.  A RTK GPS was installed on top of the tractor, see the figure below.
+
+![Phenotractor system configuration](https://github.com/Mamatemenrs/test-repo/blob/master/tractor%20setting.jpg)
+
+The distance from canopy to sensor position was measured with a sonar proximity sensor (Soutput, in mm).  Canopy height (CH) was determined by combining sonar and GPS elevation data (expressed as meter above sea level).  An elevation survey was conducted to determine a baseline reference elevation (Eref) for the gantry field.  CH was computed according to the following equation:
+
+                                               CH = Es – Eref - Soutput
+
+where  Es is sensor elevation, which was calculated by subtracting the vertical offset between the GPS antenna and sonar sensor from GPS antenna elevation.
+Infrared radiometer (IRT) sensors were used measure canopy temperature and temperature values were recoded as degree Celsius (°C).   
+Canopy spectral reflectance was measured with GreenSeeker sensors and the reflectance data were used to calculate NDVI (Normalized Difference Vegetation Index).  GreenSeeker sensors record reflected light energy in near infrared (780 ± 15 nm) and red (660 ± 10 nm ) portion electromagnetic spectrum from top of the canopy by using a self-illuminated light source. NDVI was calculated using following equation:
+
+                                           NDVI=(ρ_NIR-ρ_red)/(ρ_NIR+ ρ_red)
+                                         
+Where ρ_NIR and ρ_red represent fraction of reflected energy in near infrared and red spectral regions, respectively.
 
 - Georeferencing
 
-- Computing sensor location from sensor offset + tractor direction
+Georefencing was carried out using a specially developed Quantum GIS (GGIS, www.qgis.org ) plug-in by Andrade-Sanchez et al. (2014) during post processing. Latitude and longitude coordinates were converted to UTM coordinate system.  Offset from the sensors to the GPS position on the tractor heading were computed and corrected.  Next, the tractor data, which uses UTM Zone 12 (MAC coordinates), was transformed to EPSG:4326 (WGS84) USDA coordinates by performing  a linear shifting as follows:   
+
+                                           Latitude: Uy = My – 0.000015258894
+
+                                          Longitude: Ux = Mx + 0.000020308287 
+                                        
+where Uy and Ux are latitude and longitude in USDA coordinate system, and My and Mx are latitude and longitude in MAC coordinate system.
+Finally, georeferenced tractor data was overlaid on the gantry field polygon and mean value for each plot/genotype was calculated using the data points that fall inside the plot polygon within ArcGIS Version 10.2 (ESRI. Redlands, CA).
 
-- Calibration
 
 ## References
```

Comment:
sorry - the diff page didn't show that the ref. was still there
I'll do the latex markup, then it should be fairly easy for you to see what it looks like